src/app/beer_garden/api/http/handlers/v1/event.py

```diff
@@ -150,16 +132,24 @@ def publish(cls, event):
                         listener.request_authorization("Valid access token required")
                         continue
 
-                    if not _user_can_receive_messages_for_event(user, event):
+                    filtered_event = cls.model_filter.filter_object(
+                        obj=copy.deepcopy(event), user=user, permission="READ_ONLY"
```

Comment:
Should this inherit from the global READ_ONLY?

Reply:
Yes, we could do `permission=self.minimum_permission`, but we have to pass something. Typically we let the authentication handler figure it out, but this is the one weird scenario that does go through the router logic.